Cargo.toml

```diff
@@ -34,6 +34,9 @@ iced_wgpu = { version = "0.1.0", path = "wgpu" }
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 iced_web = { version = "0.1.0", path = "web" }
 
+[target.'cfg(target_os = "windows")'.dependencies.winapi]
+version = "0.3.6"
```

Comment:
I think this should not be necessary now.

Reply:
Oops, I forgot to remove it.
I've removed now.